shopinvader_easy_binding/models/product_template.py

```diff
@@ -34,8 +34,14 @@ def _compute_company_shopinvader_published(self):
             )
 
     def _inverse_company_shopinvader_published(self):
+        params = self.env.context.get("params", {})
+        # Sadly setting the ctx key on the flag won't work :/
+        bind_immediately = (
+            params.get("model") == "product.template"
+            and params.get("view_type") == "form"
+        ) or self.env.context.get("bind_immediately")
```

Comment:
```suggestion
        bind_immediately = self.env.context.get(
            "bind_immediately",
            self.env.context.get("params", {}).get("view_type") == "form"
        )
```
What about this?

1. It gives priority to the `bind_immediately` context key (imagine you want to set it to false, even though it's a form)
2. Applies to any form view, not just `product.template` (imagine possible related fields).. Unless of course this key is also present for imports, then no.. 

Also I think the use case for `product.product` form is missing, no?
https://github.com/shopinvader/odoo-shopinvader/blob/24446248cb51d8c3d666bc1324f1b6310a2921eb/shopinvader_easy_binding/models/product_product.py#L40

Since the `product.template` will simply delegate to the `product.product` inverse method, you could move this code there and it should work for both cases I believe.
If you do that and you decide to keep the `params.get("model") == "product.template"` mentioned above (2), check for both `product.template` and `product.product` then, as it could be triggered from either form